collect_app/src/main/java/org/odk/collect/android/formmanagement/metadata/FormMetadataParser.kt

```diff
@@ -30,6 +30,7 @@ object FormMetadataParser {
         val base64RsaPublicKey = submission?.getAttributeValue(null, "base64RsaPublicKey")
         val autoDelete = submission?.getAttributeValue(null, "auto-delete")
         val autoSend = submission?.getAttributeValue(null, "auto-send")
+        val clientEditable = submission?.getAttributeValue(null, "client-editable")
```

Comment:
I think we should convert from a String to a Boolean here (`true` if "true", `false` otherwise) which allows it to be passed around as a `Boolean` instead of a `String?` and should simplify usages.

Reply:
In the database, we store values like `autoDelete`, `autoSend`, and `clientEditable` as strings. They're parsed into booleans after being read from the database. 

In this case, we don’t even need it to be a boolean, since its only purpose is to help build the form that will eventually be saved to the database and as I mentioned, it needs to be a string at that point anyway.